nautobot/docs/installation/tables/v2-database-behavior-changes.yaml

```diff
@@ -11,9 +11,21 @@
 - Model: "Device"
   Field: "location"
   Changes: "No longer optional/nullable, must now be set"
+- Model: "Device"
+  Field: "primary_ip4"
+  Changes: "No longer one-to-one, now one-to-many (`ForeignKey`). Remote side is now a list."
+- Model: "Device"
+  Field: "primary_ip6"
+  Changes: "No longer one-to-one, now one-to-many (`ForeignKey`). Remote side is now a list."
 - Model: "IPAddress"
   Field: "role"
   Changes: "Changed from `CharField` to a `ForeignKey` to the new `Role` model."
+- Model: "IPAddress"
+  Field: "primary_ip4_for"
+  Changes: "Now a list as the reverse relation for the `Device.primary_ip4_for` foreign key."
+- Model: "IPAddress"
+  Field: "primary_ip6_for"
+  Changes: "Now a list as the reverse relation for the `Device.primary_ip6_for` foreign key."
```

Comment:
I presume these changes are part of Hanlin's branch? Since this PR only touches the VM side.

Reply:
Yeah this PR is into his branch as well.